dev/cnf/gradle/scripts/fat.gradle

```diff
@@ -213,15 +214,21 @@ task autoFVT {
       into new File(autoFvtDir, 'lib')
     }
     
-    // Copy all non-java app resourecs, such as *.html or *.jsp
+    // Copy all non-java app resources, such as *.html or *.jsp
     copy {
       includeEmptyDirs = false
       from projectDir
       into autoFvtDir
       include 'test-applications/**', 'test-bundles/**', 'test-resourceadapters/**'
       exclude '**/*.java'
     }
-    
+
+    // Copy any locally cloned git repositories
+    copy {
+      from new File(project.buildDir, 'gitRepos')
```

Comment:
this mechanism concerns me, as we'll be packing entire git repos into an autoFVT.zip.  Could this instead be done at runtime?

Reply:
This was the resolved approach agreed with the CSI squad, they do not plan to
open up GIT on the environments where the test buxkets run.
The microprofile git repos are not large as they do not contain implementations.